### ISSUE TITLE
echogenic fetal Colon

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10276]
**Preferred term label:**

**Synonyms**
Colonic hyperechogenicity
Hyperechogenic colon

**Definition (free text, please give PubMed ID)**
PMID: 17089441 In these three cases, prenatal ultrasonography demonstrated abnormal hyperechoic colonic content, which closely resembled the radiologic appearance of a barium enema(...) . Cystine accumulates in the colon and the high colonic concentration of cystine could account for the hyper-echogenic colon pattern. Moreover, in cystinuria, the small intestine transport mechanism for
cystine absorption is frequently defective (Knoll et al., 2005). (...)
Echogenic colonic meconium has previously been reported in the prenatal literature and was considered
a normal variant during the third trimester (Paulson and Hertzberg, 1991; Fung et al., 1992). (...) In conclusion, luminous echogenic colon pattern seems to be correlated with cystinuria.

PMID: 22028043 (...) involving 16 cases with prenatally detected hyperechoic colon, confirms the association between
hyperechoic colon and cystinuria when the hyperechoic colon is observed before 36 weeks’ gestation (positive predictive value, 89%). Hyperechoic colon at third trimester examination has previously been described as a normal variant because in most cases of cystinuria the clinical onset occurs after the first decade of age, meaning that most sonographers are not aware of this condition4,5.
However, when this sign was observed after 36 weeks, none of the reported cases presented with cystinuria.

PMID: 36482660 Cystinuria is an inherited disease caused by a defect in renal and intestinal tubular transport affecting cystine and dibasic amino acids (lysine,  ornithine and arginine). It is transmitted as an autosomal recessive disease. On fetal ultrasound, the colon is usually seen as hypoechoic or  isoechoic. Antenatal hyperechoic appearance of the fetal colon was previously considered as a normal variant. However, recent studies have shown that hyperechoic colon is associated with cystinuria.

**Parent term (use hpo.jax.org/app)**
Abnormal fetal gastrointestinal system morphology HP:0034207

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cystinuria  ORPHA:214

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
This seems to be the same as 
[Echogenic fetal bowel HP:0010943](https://hpo.jax.org/app/browse/term/HP:0010943)
thoughts?

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000916
- label: Echogenic fetal colon
- parents: [Echogenic fetal bowel (HP:0010943)](https://hpo.jax.org/app/browse/term/HP:0010943)
- pmid: PMID:17202504
- def: A type of echogenic fetal bowel in which the hyperechogenicity is strictly limited to the colon.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10276
- synonyms:
